condor-ap/condor_meter

```diff
@@ -893,6 +897,68 @@ def htcondor_configured():
     return False
 
 
+def get_collector_host():
+    """
+    Query for COLLECTOR_HOST
+    """
+    try:
+        path = GratiaCore.Config.getConfigAttribute("CondorLocation")
+        if g_probe_config is not None and 'htcondor-ce' in g_probe_config:
+          #condor_config_binary = 'condor_ce_config_val'
+          return None
+        else:
+          condor_config_binary = 'condor_config_val'
+        condor_config_path = os.path.join(path, "bin", condor_config_binary)
+        condor_config_val_args = ['COLLECTOR_HOST']
+
+        if (os.path.exists(condor_config_path) and
+            os.path.isfile(condor_config_path)):
+            args = [condor_config_path] + condor_config_val_args
+            cmd = subprocess.Popen(args, stdout=subprocess.PIPE)
```

Comment:
Aren't we requiring htcondor python bindings already.  Why not use the python bindings to the the config value?

Reply:
That's a fair question - I am just following suit with the other parts of the probe that use the CLI tools.

Reply:
I vote using the python API.

Reply:
@djw8605 I agree but I'd prefer that we address that in a separate PR

Reply:
It can be iterative.  Use python API for this call now.  And a separate PR to fix everywhere else...

Reply:
Ya, I'd rather get this in. We'll tackle the CLI vs python bindings usage afterwards